.golangci.yml

```diff
@@ -5,12 +5,17 @@ linters:
     - exhaustive
     - exportloopref
     - revive
-    # - goimports
+    - goimports
```

Comment:
thanks 👍 , these are just the same configs as in `chainlink` repo i'm guessing? wonder if it's overkill to include the config in the github action

Reply:
by the way, what code editor do you use? is your code editor able to use golangci-lint ? i guess that's one reason to keep it as is